### ISSUE TITLE
Use a stable version of Google Benchmark

### ISSUE THREAD

**Issue by tgonzalez89-intel** [intel/he-toolkit#35]
Build fails because of updates in the main branch of Google Benchmark.
In file _he-samples/cmake/gbenchmark.cmake_, change GBENCHMARK_GIT_TAG from _main_ to a stable release. v1.5.6. works.

Here is the exact error when installing gbenchmark:
```
[100%] Built target benchmark_main
[ 35%] Performing install step for 'ext_gbenchmark'
-- Performing Test HAVE_CXX_FLAG_WSHORTEN_64_TO_32 - Failed
-- Performing Test HAVE_CXX_FLAG_FSTRICT_ALIASING
[ 90%] Built target benchmark
[100%] Built target benchmark_main
Install the project...
-- Install configuration: "Release"
-- Installing: /home/user/he-toolkit/he-samples/build/lib/libbenchmark.a
-- Installing: /home/user/he-toolkit/he-samples/build/lib/libbenchmark_main.a
-- Installing: /home/user/he-toolkit/he-samples/build/include/benchmark
-- Installing: /home/user/he-toolkit/he-samples/build/include/benchmark/benchmark.h
-- Installing: /home/user/he-toolkit/he-samples/build/lib/cmake/benchmark/benchmarkConfig.cmake
-- Installing: /home/user/he-toolkit/he-samples/build/lib/cmake/benchmark/benchmarkConfigVersion.cmake
-- Installing: /home/user/he-toolkit/he-samples/build/lib/pkgconfig/benchmark.pc
-- Installing: /home/user/he-toolkit/he-samples/build/lib/cmake/benchmark/benchmarkTargets.cmake
-- Installing: /home/user/he-toolkit/he-samples/build/lib/cmake/benchmark/benchmarkTargets-release.cmake
-- Installing: /usr/local/share/doc/benchmark
CMake Error at src/cmake_install.cmake:152 (file):
  file INSTALL cannot make directory "/usr/local/share/doc/benchmark": No
  such file or directory.
Call Stack (most recent call first):
  cmake_install.cmake:47 (include)


make[3]: *** [install] Error 1
make[2]: *** [ext_gbenchmark/src/ext_gbenchmark-stamp/ext_gbenchmark-install] Error 2
make[1]: *** [CMakeFiles/ext_gbenchmark.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

**Comment by jlhcrawford** [comment]
Thank you for pointing this out. I will update the tag to use the stable release.

**Comment by jlhcrawford** [comment]
PR has been merged to change the Git tag from `main` to `v1.5.6`.
